whois-nrtm4/src/main/java/net/ripe/db/nrtm4/NrtmConstants.java

```diff
@@ -4,4 +4,7 @@ public class NrtmConstants {
 
     public static final int NRTM_VERSION = 4;
 
+    public static final String DELTA_PREFIX = "nrtm-delta";
```

Comment:
Create an `enum` for all possible prefix values? Or include in a per-filetype enum?